finish_release.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+. ./release.sh
+
+merge_release_candidate (){
+    echo "Merge release-candidate into release"
+    git checkout -t origin/release
+    git merge origin/release-candidate
+    git push
+}
+
+tag_release () {
+    echo "Tag release..."
+    git tag -a -m "Release $VERSION" v$VERSION
+    git push --follow-tags
+}
+
+merge_release () {
+    echo "Merge release to master"
+    git checkout -q master
+    git pull
+    git merge release
+    git push
+}
+
+main () {
+    validate_dependencies
+    create_working_dir
+    merge_release_candidate
+    tag_release
+    merge_release
+    delete_working_dir
```

Comment:
ok i see now. i think part of my confusion was that `create_working_dir` doesn't just create the directory, it also cd's into that directory